lib/validation/schema/specVersion/2.0/kind/project.json

```diff
@@ -0,0 +1,310 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema",
+	"$id": "http://ui5.sap/schema/specVersion/2.0/kind/project.json",
+
+	"type": "object",
+	"required": ["specVersion", "type", "metadata"],
+	"properties": {
+		"specVersion": { "enum": ["2.0"] },
+		"kind": {
+			"enum": ["project", null]
```

Comment:
What is the enum entry "null" for?

Reply:
The default when it's not defined. I've added comments.